### ISSUE TITLE
Sequential Distribution

### ISSUE THREAD

**Issue by geraintpalmer** [CiwPython/Ciw#103]
Add a distribution that simply returns the next value in a list/generator.

e.g.

```
'Service_distributions`: ['Sequential`, [0.1, 0.2, 0.5, 0.3]]
```

would yield:

```
>>> Q.service_times[0][0]()
0.1
>>> Q.service_times[0][0]()
0.2
>>> Q.service_times[0][0]()
0.5
>>> Q.service_times[0][0]()
0.3
>>> Q.service_times[0][0]()
0.1
>>> Q.service_times[0][0]()
0.2
```

This could be used to exactly simulate something that occurred in real life, where times where exactly recorded.

**Comment by drvinceknight** [comment]
Making it cycle would give the same functionality with added flexibility.

**Comment by geraintpalmer** [comment]
> Making it cycle would give the same functionality with added flexibility.

Yes I meant to say this. Thanks! The example shows cycling functionality.

**Comment by drvinceknight** [comment]
Cool, `itertools.cycle` will be what you want.

**Comment by geraintpalmer** [comment]
> Cool, `itertools.cycle` will be what you want.

Awsome thanks 😁

**Comment by geraintpalmer** [comment]
Include doc on sequential dist in #106 once #105 is merged in.

**Comment by geraintpalmer** [comment]
Implemented in fb461ca70f921b1e0519776ea9005d9dc048ea3f
